### ISSUE TITLE
Compability issue 

### ISSUE THREAD

**Issue by Ognyan1987** [bayaraa/d2gl#131]
When i set the game to compability win 95 or 98  i get this error

![compa](https://github.com/bayaraa/d2gl/assets/59932856/371ed2d4-6ea6-4639-8d42-701fbf83f676)

if i set the game to compabilty mode   win xp sp 3 or above all is good

**Comment by bayaraa** [comment]
Yeah it's disabled 95, 98, NT4 compat mode.
Is there any reason to use old compat modes?

**Comment by Ognyan1987** [comment]
@bayaraa   yes , Sir there are alot of reasons i do use  95/98  or XP SP 3  , map hack and loader are 2 of the reasons
Please i need 95 or 98  and Win xp SP 3  ! Or etleast enable them please in the latest and new versions of the glide  ! 

I am sure they worked with an older version i used but cannot rememmber which one  atleast it worked fine on 95/98 but i miss the new fetures and improvements from your latest releases!

**Comment by Ognyan1987** [comment]
@bayaraa  This is the error i get on  when Save and Exit from a game , diablo 2   version 1.13d online  started with out compability settings 
with the latest version of D2GL

[Error crash with no compability.txt](https://github.com/bayaraa/d2gl/files/13979296/Error.crash.with.no.compability.txt)

And this is the error i get when Save and Exit from a game  , diablo 2   version 1.13d  Online started with compability mode Win xp sp 3 , with the latest version of D2GL

[Error crash Win xp sp 3.txt](https://github.com/bayaraa/d2gl/files/13979249/Error.crash.Win.xp.sp.3.txt)

Here is what appears inside t the log,txt -> 18:39:06.639	1	8	2	524288	..\Source\D2Common\Logging\Logging.cpp	00245	Log Started

With an older Version of D2GL i am able to use comability mode WIn 95/98 with out any crashes , however setting the game to Win xp SP 3 or beyond still crashes on save and exit !

**Comment by bayaraa** [comment]
My game crashing upon start when i use 95/98/Me compat mode on win10.
Anyway, these area compat mode check removed dlls.
[wrappers.zip](https://github.com/bayaraa/d2gl/files/13984491/wrappers.zip)

**Comment by malvarenga123** [comment]
> Sir there are alot of reasons i do use 95/98 or XP SP 3 , map hack and loader are 2 of the reasons

You don't need compatibility mode to use the latest version of BH (maphack).

**Comment by Ognyan1987** [comment]
> My game crashing upon start when i use 95/98/Me compat mode on win10. Anyway, these area compat mode check removed dlls. [wrappers.zip](https://github.com/bayaraa/d2gl/files/13984491/wrappers.zip)

Ok i replaced the files u gave me  ,on Win 98/95 now is fine i do not get any Compability Warnings  and game loads perfectly , and the Save and Exit   does not cause crash anymore ! 

I still crash on Save and Exit from the game while using Win Xp Sp 3 

[Crash Win xp sp 3 online (d2log).txt](https://github.com/bayaraa/d2gl/files/14002027/Crash.Win.xp.sp.3.online.d2log.txt)
[Crash Win xp sp 3 online.txt](https://github.com/bayaraa/d2gl/files/14002028/Crash.Win.xp.sp.3.online.txt)

Also crashing on Save And exit when i disable compability at all 

[Crash  no compability online. .txt](https://github.com/bayaraa/d2gl/files/14002048/Crash.no.compability.online.txt)
[Crash  no compability online. (d2log).txt](https://github.com/bayaraa/d2gl/files/14002049/Crash.no.compability.online.d2log.txt)

Can you do a fix for that crashes or atleast for the crashes on WIn XP SP 3 , @malvarenga123  , @bayaraa  , the reasson i need WIn xp Sp 3 for D2 is that i can bind a hotkeys to my mouse buttons , in patch 1.14d this is fixed and you do not require compability , however in 1.13 it need WIn xp Sp 3 !

**Comment by bayaraa** [comment]
Probably those crashes are related to your mod or server.
There is nothing i can do further.

**Comment by malvarenga123** [comment]
@Ognyan1987 why not use Autohotkey to assign mouse buttons to keyboard hotkeys?

**Comment by Ognyan1987** [comment]
> @Ognyan1987 why not use Autohotkey to assign mouse buttons to keyboard hotkeys?

I tried , it doesnt works ingame! I assigned Mouse button 4  to "G"  by setting  XButton1::G    , started the AHK script , opened the game  , set Skill 1 to "G"  , press mouse Button 4 over a skill and nothing happened, however the  script works  while i am typing something outside of the game!

**Comment by malvarenga123** [comment]
I suggest trying the AHK forums (https://www.autohotkey.com/boards/) as this is issue is not for AHK support. I know for a fact that AHK 1.1 works in Diablo 2 (AHK needs to run with administrative privilege).
